components/card/styles.ts

```diff
@@ -8,59 +8,82 @@ export type CardStyles = {
 }
 
 export const getStyles = (
-  type: CardTypes,
+  customColor: CardColors,
   palette: ZeitUIThemesPalette,
-  isShadow?: boolean,
+  variant: CardVariants,
 ): CardStyles => {
-  const colors: { [key in CardTypes]: Omit<CardStyles, 'borderColor'> } = {
+  const solidColors: { [key in CardColors]: Omit<CardStyles, 'borderColor'> } = {
     default: {
-      color: palette.foreground,
-      bgColor: palette.background,
+      color: palette.cNeutral7,
+      bgColor: palette.cNeutral8,
     },
-    dark: {
-      color: palette.background,
-      bgColor: palette.foreground,
+    primary: {
+      color: palette.cNeutral7,
+      bgColor: palette.cTheme0,
     },
     secondary: {
-      color: palette.background,
+      color: palette.cNeutral8,
       bgColor: palette.secondary,
```

Comment:
should change the secondary here

Reply:
conflux ui keep secondary param, it is #666, we can give the url to UI and check

Reply:
remove secondary, done